src/main/java/com/patchworkmc/transformer/PatchworkTransformer.java

```diff
@@ -50,8 +55,9 @@
 	private Queue<Map.Entry<String, EventBusSubscriber>> eventBusSubscribers = new ConcurrentLinkedQueue<>(); // basename -> EventBusSubscriber
 	private Queue<Map.Entry<String, String>> modInfo = new ConcurrentLinkedQueue<>(); // modId -> clazz
 
-	public PatchworkTransformer(BiConsumer<String, byte[]> outputConsumer) {
+	public PatchworkTransformer(BiConsumer<String, byte[]> outputConsumer, IMappingProvider bridged) {
 		this.outputConsumer = outputConsumer;
+		this.bridged = bridged;
```

Comment:
`mappings` would probably be better than `bridged`

Reply:
Same as before, it has wonky behavior if duplicated names exist.